open-vulnerability-clients/src/main/java/io/github/jeremylong/openvulnerability/client/epss/EpssItem.java

```diff
@@ -26,7 +27,7 @@
  *
  * @see <a href="https://www.first.org/epss/">https://www.first.org/epss/</a>
  */
-
+@JsonIgnoreProperties(ignoreUnknown = true)
```

Comment:
```suggestion
```

Reply:
No such JSON schema exists for EPSS to my knowledge, so retained annotation as a "sensible default" for API consumption on the internet.

Reply:
no such JSON schema exists for EPSS to my knowledge so left the lenient parsing in place